starter-code/scripts/article.js

```diff
@@ -50,11 +50,21 @@ Article.fetchAll = function() {
     // When rawData is already in localStorage,
     // we can load it with the .loadAll function above,
     // and then render the index page (using the proper method on the articleView object).
-    Article.loadAll(); //TODO: What do we pass in to loadAll()?
+    Article.loadAll(JSON.parse(localStorage.rawData)); //TODO: What do we pass in to loadAll()?
+    // DONE: Actual 5 min | Expected 15 mins
     //TODO: What method do we call to render the index page?
+    // DONE: Actual 5 min | Expected 15 mins
+    articleView.initIndexPage();
   } else {
     // TODO: When we don't already have the rawData,
     // we need to retrieve the JSON file from the server with AJAX (which jQuery method is best for this?),
+    $.getJSON('/data/hackerIpsum.json').then(function(rawData){
```

Comment:
I think this is why I am getting a 404 error here. Did you look at the path? Is it correct? Your page is not loading for me.